app/onepassword_events_api/appserver/static/javascript/components/wizard.js

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from "react";
+
+import { onepassword_name_space } from "../views/setup_page";
+
+const VERSION = "1.6.0";
+const HOST = "1password.com";
+
+const e = React.createElement;
+export default function Wizard(props) {
```

Comment:
Nit: personally I think exporting named functions is nicer. It makes it less likely for the name to accidentally change (but can still be renamed if need be). I would also suggest calling it SetupWizard since component can't be re-used in the future for any other wizard flows as it currently stands.